interpreter/cling/lib/Interpreter/IncrementalParser.cpp

```diff
@@ -247,14 +247,16 @@ namespace cling {
     if (External)
       External->StartTranslationUnit(m_Consumer);
 
-    if (m_CI->getLangOpts().CPlusPlus) {
-      // <new> is needed by the ValuePrinter so it's a good thing to include it.
-      // We need to include it to determine the version number of the standard
-      // library implementation.
-      ParseInternal("#include <new>");
-      // That's really C++ ABI compatibility. C has other problems ;-)
-      CheckABICompatibility(m_CI.get());
-    }
+    // If I am the master Interpreter, only then do the #include <new>
+    if (!slaveInterp)
```

Comment:
Can you add `{` for this `if`, or combing this condition with the next one (`CPlusPlus`)?